### PR TITLE
Modify Design of Admin Flags page

### PR DESCRIPTION
**Pull request by ananyo2012**
These are some final design changes in the Admin pages. @sonalkr132 Please review. Screenshots posted below.

### CODE REVIEW

**Comment by ananyo2012** [comment]
View of flags page
flags

**Comment by ananyo2012** [comment]
View of flags_search page
flags_search

**Comment by ananyo2012** [comment]
View of flags_edit page
flags_edit

**Comment by ananyo2012** [comment]
View of flags_edit page
flags_edit

**Comment by sonalkr132** [comment]
Is reply content truncated?

**Comment by ananyo2012** [comment]
Oh no it isn't. Just doing it.

File: app/views/new_thread_flags/_new_thread_flags.html.erb [hunk 1]
```diff
@@ -1,16 +1,10 @@
-
-    <% @new_threads.each do |new_thread| %>
+<% @new_threads.each do |new_thread| %>
+  <%if @flag = new_thread.new_thread_flags.first %>
     <tr class="destroy_flag" id="new_thread_<%=new_thread.id%>">
-      <%if @flag = new_thread.new_thread_flags.first %>
-      <td class="destroy_flag"><%= @flag.new_thread.description.split[0..10].join(" ") %></td>
+      <td class="destroy_flag"><%= @flag.new_thread.description.truncate(100) %></td>
```
Comment (ananyo2012) [inline app/views/new_thread_flags/_new_thread_flags.html.erb:4]: Description is truncated now.

File: app/views/reply_flags/_reply_flags.html.erb [hunk 2]
```diff
@@ -1,15 +1,10 @@
...
-      </tr>
-      <% end %>
-    <% end %>
-  </table>
-
-
+<% @replies.each do |reply| %>
+  <%if @flag = reply.reply_flags.first %>
+    <tr class="destroy_flag" id="reply_<%=reply.id%>">
+      <td class="destroy_flag"><%= @flag.reply.content.truncate(100) %></td>
```
Comment (ananyo2012) [inline app/views/reply_flags/_reply_flags.html.erb:4]: Reply content is truncated.

**Comment by ananyo2012** [comment]
@sonalkr132 Ready for merge!

**Pull request merged**